fix ranking

kiro5502 opened the pull request:
- fix ranking, return ranking list
- fix to show player's ranking

kiro5502 committed:
fix ranking

- fix ranking, return ranking list
- fix to show player's ranking

kahmnkk commented:
수정 필요 목록

[Controller]
- 오타 확인
- id 받아올때 httpSession 사용
- Response 보낼때 Dto 사용

[RankingDto]
- pageStartNum 확인

[RankingService]
- 함수 명명규칙 확인
- sys out 빼기
- User 반환이 아닌 새로 클래스 만들어서 email, pw 빼고 반환
- PlayerRanking 함수 제거
- Start, End가 아닌 Page 값 하나 받아와서 처리

[properties]
- localhost로 커밋한거 롤백

[Test]
- Service 파일을 빈으로 등록하여 실행
- assert 함수 실행하여 원하는 대로 작동하는지 검증

수정 부탁드립니다.

kiro5502 committed:
fix ranking

- fix RankingController
- add RankingDataDto
- fix User Class
- fix RankingService
